mkdocs_macros.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+from tabulate import tabulate
+
+# This file is for defining macros for mkdocs-macros plugin
+# Check https://mkdocs-macros-plugin.readthedocs.io/en/latest/macros/ for the details
+
+
+def format_param_type(param_type):
+    if param_type == "number":
+        return "float"
+    else:
+        return param_type
+
+
+def format_param_range(param):
+    list_of_range = []
+    if "enum" in param.keys():
+        list_of_range.append(param["enum"])
+    if "minimum" in param.keys():
+        list_of_range.append(">=" + str(param["minimum"]))
+    if "exclusiveMinimum" in param.keys():
+        list_of_range.append(">" + str(param["exclusiveMinimum"]))
+    if "maximum" in param.keys():
+        list_of_range.append("<=" + str(param["maximum"]))
+    if "exclusiveMaximum" in param.keys():
+        list_of_range.append("<" + str(param["exclusiveMaximum"]))
+    if "exclusive" in param.keys():
+        list_of_range.append("≠" + str(param["exclusive"]))
+
+    if len(list_of_range) == 0:
+        return "N/A"
+    else:
+        range_in_text = ""
+        for item in list_of_range:
+            if range_in_text != "":
+                range_in_text += "<br/>"
+            range_in_text += str(item)
+        return range_in_text
+
+
+def extract_parameter_info(parameters, namespace=""):
+    params = []
+    for k, v in parameters.items():
+        if v["type"] != "object":
+            param = {}
+            param["name"] = namespace + k
+            param["type"] = format_param_type(v["type"])
+            param["description"] = v["description"]
+            param["default"] = v["default"]
+            param["range"] = format_param_range(v)
```

Comment:
I think it would be better if the columns' headers started with a capital letter: "Name", etc.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/4656/commits/f5da8db10a6c84aed21df19763e89d75e1d44efd